cyipopt/deprecated.py

```diff
@@ -0,0 +1,114 @@
+# -*- coding: utf-8 -*-
```

Comment:
Should this be `utils.py` so we can drop other functions in it for future use?

Reply:
I agree, this is a sensible suggestion.

Reply:
Addressed in https://github.com/matthias-k/cyipopt/pull/75/commits/10bd4abc247b16f19d910695002491be45cce71a.

Reply:
Addressed in https://github.com/matthias-k/cyipopt/pull/75/commits/59b8e45efe799288970c7bc9680d5bf48872a0d9.